h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -172,79 +172,82 @@ private Frame prepareLevelOneFrame(String levelOneKey, Model[] baseModels, Frame
       return levelOneFrame;
     }
 
-  /**
```

Comment:
I recommend to hide whitespaces in the diff as this file has been reindented (indentation was broken).